static/js/components/tiles/bar_tile.tsx

```diff
@@ -200,7 +200,7 @@ export const fetchData = async (props: BarTilePropType) => {
       ? Promise.resolve(null)
       : getSeriesWithin(
           apiRoot,
-          props["parentPlaces"],
+          props["parentPlace"],
```

Comment:
nit: maybe should do `props.parentPlace` and `props.enclosedPlaceType` to avoid errors like this?

Reply:
Good call- updated